app/build.gradle

```diff
@@ -150,10 +157,13 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit5_version"
     testRuntimeOnly "org.junit.vintage:junit-vintage-engine:$junit5_version"
 
+    androidTestImplementation rootProject.ext.aosp_libs
```

Comment:
I think I can remove this...

Reply:
This *should* already get pulled in via `implementation`.

Reply:
Removed.